app/src/main/java/io/homeassistant/companion/android/widgets/camera/CameraWidget.kt

```diff
@@ -228,41 +148,60 @@ class CameraWidget : AppWidgetProvider() {
             return
         }
 
-        mainScope.launch {
+        widgetScope?.launch {
             Log.d(
                 TAG,
                 "Saving camera config data:" + System.lineSeparator() +
                     "entity id: " + entitySelection + System.lineSeparator()
             )
-            cameraWidgetDao.add(
+            repository.add(
                 CameraWidgetEntity(
-                    appWidgetId,
-                    serverSelection,
-                    entitySelection,
-                    tapActionSelection
+                    id = appWidgetId,
```

Comment:
This change feels unnecessary adds more to the amount of lines changed, we should avoid reformatting the page as it makes the review more difficult.